docs/cmdline-opts/proxy-tls13-ciphers.md

```diff
@@ -2,28 +2,32 @@
 c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 Long: proxy-tls13-ciphers
-Arg: <ciphersuite list>
+Arg: <list of ciphers>
```

Comment:
I propose just `<list>`. The documentation explains the list format and how to write it.